src/home.js

```diff
@@ -0,0 +1,18 @@
+const homePage = () => {
+  const content = document.getElementById('content');
+
+  const subHeading = document.createElement('h3');
+  subHeading.innerHTML = 'Chef Rob Hanzer perfectly cooks chickens in a giant rotisserie oven, but the birds are equally delicious roasted on a rack in a home oven.';
+
+  const desc = document.createElement('p');
+  desc.innerHTML = 'Located in the Gramercy Park Hotel, Maialino captures the warmth and comfort of a Roman trattoria, with a menu that reflects the rich cultures of both Rome and New York. <br /> Executive Chef Matt Spivey draws inspiration from the Roman cooking tradition to create soulful food that celebrates the restaurant’s ongoing relationship with its local family of farmers and suppliers.';
+
+  const image = document.createElement('img');
+  image.setAttribute('src', './roast-chicken.jpg');
+
+  content.appendChild(image);
```

Comment:
- [ ] **[OPTIOAL]** To increase performance and avoid reflows every time you are appending a child to the parent element, you can use document fragments
      Read more aobut [reflows & document fragements](https://developer.mozilla.org/en-US/docs/Web/API/Document/createDocumentFragment)